book/src/theme-gallery/index.md

```diff
@@ -114,3 +115,122 @@ It features a dominant partition of space into a bright and a dark side.
 ```
 ![bipartite theme screenshot](./bipartite.png)
 
+## University
+_[go to top](#theme-gallery)_
+
+This theme offers a simple yet versatile design, allowing for easy customization
+and flexibility. Additionally, it incorporates a progress bar at the top, which
+displays the current status of the presentation.
+
+By default, this theme utilizes multiple body attachments for every slide type,
+with each additional body creating a new column on the slide. Columns are
+equally sized by default, but you can modify their dimensions by providing a
+columns parameter to the slide or the variant.
+
+The theme supports the addition of multiple bodies on every slide type. Each
+added body creates a new column within the slide.
+Although the columns have equal dimensions by default, their sizes can be easily
+adjusted by supplying a `columns` parameter to either the slide or the variant.
+
+The `columns` parameter can be configured similarly to the `columns` grid
+parameter. For detailed instructions on customization, please consult the
+official documentation.
+
+### Theme options
+- `institution-name`: The institution name
+- `color-a`: The main color
+- `color-b`: The accent accent
+- `color-c`: A color of your choice
+- `logo`: An element that will be on the top right of the first slide. Usually
+  an image element, like a logo.
+
+### Default slide options
+
+It is possible to override the default theme option parameters by providing
+them in the `#slide` function. It is also possible to override other parameters
+for more customization options. These parameters are:
+
+- `short-authors`
+- `short-title`
+- `date`
```

Comment:
I had to use that feature recently when I was doing a presentation with someone else. I had to change the `short-authors` in the bottom left because we were presenting both the same presentation. Quite handy.

Reply:
Fair enough.